### PR TITLE
Add ClientManagementPage and ClientDetailsPage

### PR DESCRIPTION
**Pull request by MarcoBramini**
This PR adds the ClientManagementPage and ClientDetailsPage as requested in this [task](https://polito-se2-21-09.myjetbrains.com/youtrack/issue/S202109SPG-21).

The ClientOrderList is scheduled to be done in a different task (https://polito-se2-21-09.myjetbrains.com/youtrack/issue/S202109SPG-25).

### CODE REVIEW

File: client/src/services/models/ClientInfo.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class ClientInfo {
+  constructor(id, email, password, fullName, address, wallet) {
+    this.id = id;
+    this.email = email;
+    this.password = password;
```
Comment (G-Salviati) [inline client/src/services/models/ClientInfo.js]: The password doesn't exist yet

File: client/src/ui-components/ClientInfoList/ClientInfoList.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { Col, Container, Row, Spinner } from "react-bootstrap";
+import ClientInfoListHeader from "./ClientInfoListHeader";
+import ClientInfoListItem from "./ClientInfoListItem";
+
+function ClientInfoList(props) {
+  function getEmptyListText() {
+    if (props.clientInfoList === null)
```
Comment (G-Salviati) [inline client/src/ui-components/ClientInfoList/ClientInfoList.js:7]: I didn't understand this line. It involves that the clientinfolist is empty when you go to the employee management panel? and only after a search it retrieves a subset of clients?

**Review by G-Salviati — CHANGES_REQUESTED** [review]
Changes only about the password property in ClientInfo model I suppose

File: client/src/ui-components/ClientInfoList/ClientInfoList.js (hunk 2 above)
Comment (MarcoBramini) [reply client/src/ui-components/ClientInfoList/ClientInfoList.js:7]: Yes, there is no reason to preload all the users. The employee can enter the page -> write the details of the user he wants to search -> select the correct user between the items

File: client/src/services/models/ClientInfo.js (hunk 1 above)
Comment (MarcoBramini) [reply client/src/services/models/ClientInfo.js]: ops

**Review by G-Salviati — APPROVED** [review]

**Pull request merged**